### PR TITLE
bugfix for project's .asf.yaml

### PR DESCRIPTION
**Pull request by StevenLuMT**
Descriptions of the changes in this PR:

### Motivation

using [.asf.yaml](https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features#Git.asf.yamlfeatures-Branchprotection) to configure the project,
but there 's some config bugfix for project's .asf.yaml
the name of workflow's jobs is not set, so #3439 causes contexts can't match check's jobs

this picture is the other new add pr's running result: the other pr's workflow run abnormally
image

this picture is current running result: current pr is running normally because the job name is added in yaml file
image
 
### Changes

1.  add the name for checks
2. change required_approving_review_count to 2 person
3. update README.md for workflows

### CODE REVIEW

**Comment by StevenLuMT** [comment]
rerun failure checks

**Review by tisonkun — APPROVED** [review]
Great!

**Review by Shoothzj — APPROVED** [review]

**Review by zymap — APPROVED** [review]

File: .asf.yaml
Comment (eolivelli) [inline .asf.yaml:66]: This is wrong.
Otherwise committers won't be able to commit their own patches with only one approval from another committer.
The rule is that you need two +1 from people with committer karma.

So if a committer sends a patch,he needs only 1 binding +1

@StevenLuMT @zymap

Comment (tisonkun) [reply .asf.yaml:66]: Basically, I'd prefer not to set it as a strict rule since a committer should be confident to merge a typo fix. Committers share a high trust level. They know when to request one more reviewer.

Comment (StevenLuMT) [reply .asf.yaml:66]: @eolivelli  @zymap  More content can be discussed, I will initiate a discussion, let’s determine the final .asf.yaml
please see the mail  [https://lists.apache.org/thread/o75xogn8r7zylqlwnno84dsm9nqoh5rs](https://lists.apache.org/thread/o75xogn8r7zylqlwnno84dsm9nqoh5rs)

**Pull request merged**
